### ISSUE TITLE
Arbitrary Functions for Nutrients and Velocity Field

### ISSUE THREAD

**Issue by Atreyu-94** [JuliaOcean/PlanktonIndividuals.jl#73]
I know that nutrients are imposed as initial conditions. But could some variations of the nutrients be inserted during the simulation, for example through a function?

**Comment by zhenwu0728** [comment]
Yes, you can use boundary conditions to do that. The code is [here](https://github.com/JuliaOcean/PlanktonIndividuals.jl/blob/master/src/Fields/boundary_conditions.jl). Sorry it's not well documented, but let me know if you run into any problems.
